package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@webex/sdk-component-adapter",
   "version": "1.97.2",
   "main": "dist/webexSDKComponentAdapter.esm.js",
-  "module": "dist/webexSDKComponentAdapter.esm.js",
```

Comment:
Don't remove this property from `package.json`. This is meant to be picked up by tooling that supports module systems in the consumer environment of this library.